Cargo.toml

```diff
@@ -2,3 +2,18 @@
 name = "acmed-hook-rfc2136"
 version = "0.1.0"
 edition = "2021"
+
+[dependencies]
+anyhow = "1.0.71"
+base64 = "0.21.0"
+clap = { version = "4.2.7", features = ["derive"] }
+serde = { version = "1.0.163", features = ["derive"] }
+serde_with = { version = "3.0.0", features = ["base64"] }
+time = "0.3.21"
+toml = "0.7.3"
+hickory-client = { version = "0.24.0", features = ["dnssec", "dnssec-openssl"] }
+hickory-proto = { version = "0.24.0", features = ["dnssec-openssl"] }
+tracing = "0.1"
+tracing-subscriber = { version = "0.3", features = ["env-filter"] }
+once_cell = "1.18"
+backoff = "0.4"
```

Comment:
Can you sort these alphabetically? Also: *if* we're using openssl for dnssec in hickory, we should *probably* vendor the openssl stuff and statically include it. You can do that by including a line like `openssl = { version = "0.10.59", features = ["vendored"] }`